typescript/src/supernova.d.ts

```diff
@@ -287,17 +287,23 @@ declare global {
     type: DocumentationItemType
     slug: string
     userSlug: string | null
+    configuration: {
+      isHidden: boolean
+      isPrivate: boolean
+    }
   }
 
   type DocumentationGroup = DocumentationItem & {
+    type: "Group"
     isRoot: boolean
     childrenIds: Array<string>
-    children: Array<DocumentationItem>
+    children: Array<DocumentationGroup | DocumentationPage>
```

Comment:
Changing type since so we don't need to type cast it all the time. Only those 2 can be the options.